src/sage/modular/cusps.py

```diff
@@ -465,7 +466,8 @@ def __neg__(self):
         """
         return Cusp(-self.__a, self.__b)
 
-    def is_gamma0_equiv(self, other, N, transformation=None):
+    def is_gamma0_equiv(self, other, N,
+                        transformation=None) -> tuple[bool, Any]:
```

Comment:
I think from the description and code this should instead be:

```suggestion
                        transformation=None) -> bool | tuple[bool, Any]:
```
since some inputs will output only a bool.